pkg/operator/kit/resource.go

```diff
@@ -72,15 +73,45 @@ func CreateCustomResource(resource CustomResource, clientset kubernetes.Interfac
 	}
 	kubeVersion := version.MustParseSemantic(serverVersion.GitVersion)
 
-	if kubeVersion.AtLeast(version.MustParseSemantic("v1.7.0")) {
-		return createCRD(resource)
+	if kubeVersion.AtLeast(version.MustParseSemantic(serverVersionV170)) {
+		// Create and wait for CRD resources
+		apiExtClientset, err := getApiextensionsClientset()
```

Comment:
i'm suggesting this clientset should be initialized at the same level as the other clientset. Unit tests can't be created at this level if it is retrieved here. See https://github.com/kokhang/rook/blob/4424c4624f314022622c7f53e176da99b640704d/cmd/rook/operator.go#L56